### DISCUSSION TITLE
Merging `content` from presets

### DISCUSSION THREAD

**Discussion by dallonf** [tailwindlabs/tailwindcss#12394] [Ideas]
For context, my team is using Tailwind to create a shared design system for our projects, both with design tokens and common React components.

The design tokens can be easily shared with a Tailwind preset, but without configuring `content` explicitly, the React components' styles won't be picked up by the Tailwind configuration of a project that uses that design system.

It would be really nice if we could just add the paths to shared component source files to the Tailwind preset, so every project that uses it would get those styles, but according to the [documentation on merging](https://tailwindcss.com/docs/presets#merging-logic-in-depth), `content` is completely overridden by the final configuration.

**Comment by kevinruscoe** [comment]
Stumbled across this today and got it working.

Require in your config with 
```
let tailwindConfig = require('./../other-path/tailwind-preset.js');
``` 

then in the parent config use 

```
content: [
    './**/*.html',
  ].concat(tailwindConfig.content),`
```
  
That'll merge the 2 arrays together. Relative paths don't work on the preset file though, so use 

```
content: [
    __dirname + '/**/*.html',
]
``` 

to ensure the full path is used.
